docs/wiki/Whats-new.md

```diff
@@ -56,6 +56,7 @@ Here's what's changed in Enterprise Scale/Azure Landing Zones:
 - Updated "**Deploy Diagnostic Settings to Azure Services**" initiative replacing deprecated policy for diagnostic settings on Storage Account
 - Removed all exclusions (parameters) from the Microsoft Cloud Security Benchmark (currently Azure Security Benchmark) initiative assignment to standardize across reference architectures and align with best practice.
 Impacted assignment: Deploy-ASC-Monitoring
+- Removed ActivityLog Solution as an option to be deployed into the Log Analytics Workspace.
```

Comment:
```suggestion
- Removed `ActivityLog` Solution as an option to be deployed into the Log Analytics Workspace. As this has been superseded by the Activity Log Insights Workbook, as documented [here.](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-insights)
```

Reply:
@lachaves can we also move this into the correct section, instead of it being in the policy section?

`Tooling` is the correct section, which you may need to add for December

Reply:
@jtracey93, sounds good, will change now.